types/saturn-web/index.d.ts

```diff
@@ -0,0 +1,37 @@
+declare namespace Saturn {
+    interface SaturnSettings {
+        integrationId: string;
+    }
+
+    interface SaturnInitializer {
+        run: (options: SaturnSettings) => void;
+    }
+
+    interface SaturnInterface {
+        integrationId: string;
+        isOpen: boolean;
+        isLoaded: boolean;
+        isUserLoaded: boolean;
+        setUser: (
+            identifier: string,
+            user: {
+                email: string;
+                name?: string;
+                identifierHash?: string;
```

Comment:
```suggestion
                name?: string | undefined;
                identifierHash?: string | undefined;
```

For `exactOptionalPropertyTypes`.